api/src/server.ts

```diff
@@ -37,105 +37,110 @@ const rollbar = new Rollbar({
-  ttl: (24 * 60 * 60), // 24 hours in seconds
-});
-
-const sessionRepository = AppDataSource.getRepository(Session);
-app.use(
-  session({
-    name: config.auth.sessionName,
-    secret: config.auth.sessionSecret,
-    resave: false, // don't save session if unmodified
-    saveUninitialized: false, // don't create session until something stored
-    proxy: true, // trust first proxy for secure cookies in production
-    store: sessionStore.connect(sessionRepository),
-    cookie: {
-      sameSite: 'none', // required for cross-site cookies
-      secure: true, // Must be true when samesite is 'none' and using HTTPS (or with localhost)
-      httpOnly: true, // Helps prevent XSS attacks by not allowing client-side scripts to access the cookie
-    },
-  })
-);
-
-// Body parser middleware for SAML authentication
-app.use(bodyParser.urlencoded({ extended: false, limit: '10mb' }));
-app.use(bodyParser.json({ limit: '10mb' }));
-
-// Init Passport middleware
-app.use(passport.initialize());
-app.use(passport.session());
-if (config.auth.strategy === 'local') {
-  initLocalPassport(app, userService);
-}
-if (config.auth.strategy === 'saml') {
-  const samlStrategy = getSamlStrategy(userService);
-  initSamlPassport(samlStrategy, userService);
-  app.use('/api/sso', SamlAuthRoutes(samlStrategy));
-}
+  const sessionRepository = AppDataSource.getRepository(Session);
+  app.use(
+    session({
+      name: config.auth.sessionName,
+      secret: config.auth.sessionSecret,
+      resave: false, // don't save session if unmodified
+      saveUninitialized: false, // don't create session until something stored
+      proxy: true, // trust first proxy for secure cookies in production
+      store: sessionStore.connect(sessionRepository),
+      cookie: {
+        sameSite: 'none', // required for cross-site cookies
+        secure: true, // Must be true when samesite is 'none' and using HTTPS (or with localhost)
+        httpOnly: true, // Helps prevent XSS attacks by not allowing client-side scripts to access the cookie
+      },
+    })
+  );
+
+  // Body parser middleware for SAML authentication
+  app.use(bodyParser.urlencoded({ extended: false, limit: '10mb' }));
+  app.use(bodyParser.json({ limit: '10mb' }));
+
+  // Init Passport middleware
+  app.use(passport.initialize());
+  app.use(passport.session());
+  if (config.auth.strategy === 'local') {
+    initLocalPassport(app, userService);
+  }
+  if (config.auth.strategy === 'saml') {
+    const samlStrategy = getSamlStrategy(userService);
+    initSamlPassport(samlStrategy, userService);
+    app.use('/api/sso', SamlAuthRoutes(samlStrategy));
+  }
+
+  // Register  unprotected routes
+  app.get('/api', (req: Request, res: Response) => {
+    res.json({
+      success: true,
+      isAuthenticated: req.isAuthenticated(),
+      message: 'DMSP AI Tool API',
+    });
+  });
+  if (config.auth.strategy === 'local') {
+    app.use('/api/auth', LocalAuthRoutes());
```

Comment:
I hadn't noticed this previously, but we should be able to merge this block with the local strategy block above on line 102.